common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -24,24 +24,21 @@ object ImageMetadataConverter {
       .distinct
   }
 
-  private def extractPeople(fileMetadata: FileMetadata): List[String] = {
-    val xmpIptcPeople = fileMetadata.xmp.filterKeys(_ matches "Iptc4xmpExt:PersonInImage\\[\\d+\\]")
-      .values
-      .toList
-      .flatMap{
-        case JsString(value) => Some(value)
-        case _ => None
-      }
+private def extractXMPArrayStrings(field: String, fileMetadata: FileMetadata): Seq[String] =
```

Comment:
Woah! This should allow us to read [keywords from XMP arrays](https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala#L69-L70)!